xskillscore/tests/test_comparative.py

```diff
@@ -220,3 +233,101 @@ def test_sign_test_NaNs_confidence(a, a_worse, b):
+            a_1d_worse_less_corr["var2"] = a_1d_worse_less_corr["var"] * 2
+            b_1d["var2"] = b_1d["var"] * 2
+    elif input == "mixed":
+        name = "var"
+        a_1d = a_1d.to_dataset(name=name)
+    if chunk:
+        a_1d = a_1d.chunk()
+        a_1d_worse_less_corr = a_1d_worse_less_corr.chunk()
+        b_1d = b_1d.chunk()
+    actual_diff, actual_alpha = mae_test(a_1d, a_1d_worse_less_corr, b_1d, alpha=alpha)
+    # check dask collection preserved
+    for actual in [actual_diff, actual_alpha]:
+        assert is_dask_collection(actual) if chunk else not is_dask_collection(actual)
+
+
+@pytest.mark.parametrize("alpha", [0.0, 0, 1.0, 1.0, 5.0, 5])
+def test_mae_test_alpha(a_1d, a_1d_worse_less_corr, b_1d, alpha):
+    """Test mae_test alpha error messages."""
+    with pytest.raises(ValueError) as e:
+        mae_test(a_1d, a_1d_worse_less_corr, b_1d, alpha=alpha)
+    assert "`alpha` must be between 0 and 1 or `return_p`" in str(e.value)
+
+
+@pytest.mark.parametrize("alpha", [0.05, "return_p"])
+def test_mae_test_better_significant(a_1d, a_1d_worse_less_corr, b_1d, alpha):
+    """Test mae_test favors better forecast."""
+    a_1d_worse_less_corr = a_1d.copy()
+    # make a_worse worse every second timestep
+    step = 3
+    a_1d_worse_less_corr[::step] = a_1d_worse_less_corr[::step] + OFFSET * 3
+    actual_diff, actual_alpha = mae_test(a_1d, a_1d_worse_less_corr, b_1d, alpha=alpha)
+    if alpha != "return_p":
+        assert (actual_diff > actual_alpha).all()
+    else:
+        assert actual_alpha < 0.05
+
+
+def test_mae_test_climpred(a_1d, b_1d):
+    """Test mae_test as climpred would use it with observations=None."""
+    a_1d_worse_less_corr = a_1d.copy()
+    # make a_worse worse every second timestep
+    a_1d_worse_less_corr[::2] = a_1d_worse_less_corr[::2] + OFFSET
+    # calc skill before as in climpred
+    dim = []
+    time_dim = "time"
+    mae_f1o = mae(a_1d, b_1d, dim=dim)
+    mae_f2o = mae(a_1d_worse_less_corr, b_1d, dim=dim)
+
+    actual_diff, actual_alpha = mae_test(
+        mae_f1o,
+        mae_f2o,
+        observations=None,
+        dim=dim,
+        time_dim=time_dim,
+    )
+    expected_diff, expected_alpha = mae_test(
+        a_1d, a_1d_worse_less_corr, b_1d, dim=dim, time_dim=time_dim
+    )
+    assert_equal(actual_diff, expected_diff)
+    assert_equal(actual_alpha, expected_alpha)
+
+
+@pytest.mark.parametrize("dim", [[], ["lon", "lat"]])
+def test_mae_test_dim(a, b, dim):
+    """Test mae_test for different dim on ."""
+    a_worse = a.copy()
+    # make a_worse worse every second timestep
+    a_worse[::2, :, :] = a_worse[::2, :, :] + OFFSET
+    actual_diff, actual_alpha = mae_test(a, a_worse, b, dim=dim)
+    # difference larger than half width ci
+    assert (actual_diff > actual_alpha).mean() > 0.5
+    for d in dim:
+        assert d not in actual_diff.dims, print(d, "found, but shouldnt")
+
+
+def test_mae_test_alpha_hwci(a_1d, a_1d_worse_less_corr, b_1d):
```

Comment:
Might be worth adding a test for symmetry. I.e. that the order of forecasts input doesn't change the answer, only the sign in the difference.

Reply:
`pearson_r` and `mae` are symmetric themselves